client/components/notice/notice-action.jsx

```diff
@@ -23,6 +24,12 @@ export default React.createClass( {
 		};
 	},
 
+	componentDidMount() {
+		if ( this.props.onMount ) {
+			this.props.onMount();
```

Comment:
Is there any other way we can handle this? Seems like a potential avenue for problematic usage.